src/index.d.ts

```diff
@@ -3768,6 +3768,28 @@ declare namespace esb {
         spanQry?: SpanQueryBase
     ): SpanFieldMaskingQuery;
 
+    export class SparseVectorQuery extends Query {
```

Comment:
suggestion: I know this is a bit tedious but could you repeat the documentation comments here for the exported class, function and function methods?